.config/nextest.toml

```diff
@@ -9,4 +9,4 @@ slow-timeout = { period = "30s", terminate-after = 4 }
 [profile.ci]
 fail-fast = true
 failure-output = "immediate"
-slow-timeout = { period = "30s", terminate-after = 2 }
+slow-timeout = { period = "30s", terminate-after = 6 }
```

Comment:
The longest tests before took a little over 40s out of 60s already. Now it takes 80s. I wanted to have at least twice the headroom, before it seemed like it might cause flakes.

Reply:
I cannot answer that. We make one call per peer at the beginning of the test, so one would expect it to be slower however not 40s. IMO there’s something wrong with the testing framework; as I said the mint tests were already before close to time out at 40s.

Reply:
Will bandaid it with https://github.com/fedimint/fedimint/pull/5300 for now.